governance/xc_admin/packages/xc_admin_frontend/components/programs/PythCore.tsx

```diff
@@ -400,7 +399,10 @@ const PythCore: React.FC<PythCoreProps> = ({ proposerServerUrl }) => {
 
   useEffect(() => {
     if (!dataIsLoading && rawConfig) {
-      const downloadableConfig = getDownloadableConfig(rawConfig)
+      const downloadableConfig = getDownloadableConfig(rawConfig) as Record<
+        string,
+        DownloadableProduct
+      >
```

Comment:
Why do we need a typecast here?  If the return types of `getDownloadableConfig` are accurate, we shouldn't need the typecast.  If they aren't accurate, then we shouldn't be typecasting, we should be fixing the types to be accurate.